gsa/src/web/pages/reports/listpage.js

```diff
@@ -194,8 +194,8 @@ class Page extends React.Component {
   }
 
   handleReportDeleteClick(report) {
-    const {gmp, onChanged, onError} = this.props;
-    return gmp.report.delete(report).then(onChanged, onError);
+    const {onDeleted} = this.props;
+    return onDeleted(report);
```

Comment:
Should be named `onDelete` because we are going to delete something and aren't calling this function after something has been deleted.
```suggestion
    return onDelete(report);
```

Reply:
Good point. Will do.